Adding UCL examples & editorial updates (#3)
boucadair/policy-based-network-acl: draft-ma-opsawg-ucl-acl.md

--- back

# Example Usage

## Configuring the Controller Using Group based ACL {#controller-ucl}

>    Assume the organization would like to restrict the access of R&D

Reviewer: ```suggestion
   Let's consider an organization that would like to restrict the access of R&D
```

>    employees that bring personally owned devices into the workplace.

Reviewer: ```suggestion
   employees that bring personally owned devices (BYOD) into the workplace.
```

>    Access requirements are following:

Reviewer: ```suggestion
   The access requirements are as follows:
```

>    * Permit traffic from R&D BYOD employees, destined to R&D employees.

Reviewer: ```suggestion
   * Permit traffic from R&D BYOD of employees, destined to R&D employees' devices.
```

>    * Deny traffic from R&D BYOD employees, destined to finance servers

Reviewer: ```suggestion
   * Deny traffic from R&D BYOD of employees, destined to finance servers
```

     located in the enterprise DC network.

>    The following example illustrates configuring the SDN controller

Reviewer: ```suggestion
   The following example illustrates the configuration of the SDN controller
```

   using the group-based ACL:

~~~~
{::include ./examples/controller-ucl.xml}
~~~~

## Configuring the PEP Using Group based ACL {#PEP-ucl}

>    This section illustrates an example configuring the PEP device using

Reviewer: ```suggestion
   This section illustrates an example to configure a PEP  using
```

   the group-based ACL.

   The PEP which enforces group-based ACL may acquire group identities
   from the AAA server if working as a NAS authenticating both the
   source endpoint and the destination endpoint users. Another case for
   a PEP enforcing a group-based ACL is to obtain the group identity of
   the source endpoint directly from the packet field
   {{?I-D.smith-vxlan-group-policy}}. This does not intend to be exhaustive.

   Assume the mapping between device group ID and IP addresses is
   predefined or acquired via device authentication. The following example
   shows ACL configurations delivered from the controller to the PEP. This
   example is consistent with the example presented in {{controller-ucl}}.

~~~~
{::include ./examples/PEP-ucl.xml}
~~~~

## Configuring the PEP Using Address based ACL {#PEP-acl}

>    The section illustrates an example configuring the PEP device using

Reviewer: ```suggestion
   The section illustrates an example of configuring a PEP using
```
